### PR TITLE
Contributing test coverage added missing directory

### PR DESCRIPTION
**Pull request by jennuine**
Signed-off-by: Jenn Nguyen <jenn@openrobotics.org>

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -459,13 +459,13 @@ coverage report. You'll need to have [lcov](http://ltp.sourceforge.net/coverage/
 
 1. In your `build` folder, compile with `-DCMAKE_BUILD_TYPE=Coverage`
 
-        cmake -DCMAKE_BUILD_TYPE=Coverage ..\
+        cmake -DCMAKE_BUILD_TYPE=Coverage ../
```
Comment (chapulina) [inline contributing.md:462]: It looks like the `colcon` vs `cmake` difference made another victim, sorry about that :confused: 

See another related PR: https://github.com/ignitionrobotics/docs/pull/40

Your change on line 467 suggests you're in a colcon workspace. Where are you running this command in that case?

Comment (jennuine) [reply contributing.md:462]: Yes, I was a bit confused at first. I'm using a colcon workspace. I also tested the `cmake` command in a build folder in the source directory of the package. Should I make this more clear in the doc?

Comment (mjcarroll) [reply contributing.md:462]: I don't believe that the slash should be necessary in either case, correct?

Comment (jennuine) [reply contributing.md:462]: No it's not necessary

Comment (chapulina) [reply contributing.md:462]: > Should I make this more clear in the doc?

Yeah I think we can err on the side of being more verbose here, but cover all use cases.

So we should explain that, if the user is in a colcon workspace, they should do

```
cd <path to colcon workspace>
colcon build --cmake-args -DCMAKE_BUILD_TYPE=Coverage
```

And if they're using plain cmakle

```
cd <path to build directory>
cmake -DCMAKE_BUILD_TYPE=Coverage ..
```

Comment (jennuine) [reply contributing.md]: Made clarification changes here 14dd9e53b57ead6f9584d971a495dfb4bcd16c2a

File: contributing.md [hunk 2]
```diff
@@ -459,13 +459,32 @@ coverage report. You'll need to have [lcov](http://ltp.sourceforge.net/coverage/
...
-       ./workspace/build/package_name/bin/UNIT_TestName_TEST   (single test)
-        make test                                          (all tests)
+    If using a colcon workspace:
+
+       # single test
+       ./build/<package_name>/bin/UNIT_<TestName>_TEST
+
+       # all tests
+       cd build/<package_name>
+       make test
```
Comment (chapulina) [inline contributing.md:482]: This works, but I think that when using colcon, this is cleaner:

```
        cd <path to colcon workspace>
        colcon test --packages-select <package_name>
```

This way the logs end up in the `logs` directory.

Comment (jennuine) [reply contributing.md:482]: ok, I'll update that. Is there a `colcon` command to run all tests? The command you provided only runs one single test?

Comment (chapulina) [reply contributing.md:482]: Here's a quick cheat sheet:

Run all tests for all packages in the workspace:

```
colcon test
```

Run all tests for a single package:

```
colcon test --packages-select <package_name>
```

Run specific tests:

```
colcon test --ctest-args -R <part of test name>
# or
./build/<package_name>/bin/<TestName>
```

**Review by chapulina — APPROVED** [review]
Looks good! Thanks for iterating.

**Pull request merged**